ui/apps/platform/src/Containers/Clusters/ClusterSummaryLabelsConfiguration.tsx

```diff
@@ -0,0 +1,109 @@
+    centralVersion,
+    clusterRetentionInfo,
+    selectedCluster,
+    managerType,
+    handleChange,
+    handleChangeAdmissionControllerEnforcementBehavior,
+    handleChangeLabels,
+}: ClusterSummaryLabelsConfigurationProps): ReactElement {
+    const isManagerTypeNonConfigurable =
+        managerType === 'MANAGER_TYPE_KUBERNETES_OPERATOR' ||
+        managerType === 'MANAGER_TYPE_HELM_CHART';
+
+    return (
+        <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsMd' }}>
+            {/* @TODO, replace open prop with dynamic logic, based on clusterType */}
+            {selectedCluster.id && selectedCluster.healthStatus ? (
+                <ClusterSummaryLegacy
+                    healthStatus={selectedCluster.healthStatus}
+                    status={selectedCluster.status}
+                    centralVersion={centralVersion}
+                    clusterId={selectedCluster.id}
+                    autoRefreshEnabled={selectedCluster.sensorCapabilities?.includes(
+                        'SecuredClusterCertificatesRefresh'
+                    )}
+                    clusterRetentionInfo={clusterRetentionInfo}
+                    isManagerTypeNonConfigurable={isManagerTypeNonConfigurable}
+                />
+            ) : (
+                <Alert variant="warning" isInline title="Legacy installation method" component="p">
+                    <Flex direction={{ default: 'column' }}>
+                        <FlexItem>
+                            <p>
+                                To avoid extra operational complexity, use a{' '}
+                                <strong>cluster init bundle</strong> with either of the following
+                                installation methods:
+                            </p>
+                            <p>
+                                <strong>Operator</strong> for Red Hat OpenShift
+                            </p>
+                            <p>
+                                <strong>Helm chart</strong> for other platforms
+                            </p>
+                        </FlexItem>
+                        <FlexItem>
+                            <p>
+                                Only use the legacy installation method if you have a specific
+                                installation need that requires using this method.
+                            </p>
+                        </FlexItem>
+                    </Flex>
+                </Alert>
+            )}
+            <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsSm' }}>
+                <Title headingLevel="h2">Cluster labels</Title>
+                <ClusterLabelsTable
+                    labels={selectedCluster?.labels ?? {}}
+                    handleChangeLabels={handleChangeLabels}
+                    hasAction
+                />
+            </Flex>
+            <Flex direction={{ default: 'column' }} spaceItems={{ default: 'spaceItemsSm' }}>
```

Comment:
nit: I feel like the sections are a bit compact, so it's hard to visually assess the separation.

Between `Cluster labels` and `Static configuration`
<img width="1317" height="294" alt="Image" src="https://github.com/user-attachments/assets/7f0c8f00-2560-4227-a679-c439a2443ab1" />

Between `Static configuration` and `Dynamic configuration`
<img width="1309" height="262" alt="Image" src="https://github.com/user-attachments/assets/93a667b6-2137-4628-8730-ba09c25270eb" />